### ISSUE TITLE
Show cancelled status for course sections

### ISSUE THREAD

**Issue by galacticglum** [sqrl-planner/sqrl-client#109]
If a section has been cancelled (``cancelled`` attribute on the Section returned by gator), we should show the section as cancelled on the client, make it disabled (can't add it to timetable).

We might also want to notify users who have added the cancelled section to their timetable.

**Comment by eamonma** [comment]
Notifications would necessitate a more elaborate discussion, regarding the scope, implementation, and service provider. I think we can hold off on that part for now.

**Comment by galacticglum** [comment]
Agreed. I mostly just wanted to have a "paper trail" for that idea so we don't forget it. Someone (or I can) should open a ticket for it though.

**Comment by eamonma** [comment]
We can expand the scope of #80 to adding notifications in general.

**Comment by eamonma** [comment]
To flesh this one out more, if someone has already added it to the timetable, it should be shown as cancelled in the overview, in the style similar to a no credit course such as PDC, and not counted as credit.
